### PR TITLE
Add notes about behavior changes to model binding in 5.0

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: aspnetcore/migration/31-to-50.md [hunk 1]
```diff
@@ -120,3 +120,24 @@ As part of the move to ".NET" as the product name, the Docker images moved from
 ## Review breaking changes
 
 For breaking changes from .NET Core 3.1 to .NET 5.0, see [Breaking changes for migration from version 3.1 to 5.0](/dotnet/core/compatibility/3.1-5.0). ASP.NET Core and Entity Framework Core are also included in the list.
+
+## Changes to model binding in ASP.NET Core MVC and Razor Pages
```
Comment (pranavkm) [inline aspnetcore/migration/31-to-50.md:124]: I don't think these qualify as breaking changes but it is useful to document these in some place and point users to if they ask.

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@pranavkm  see my commit for slight tweaks.

**Review by pranavkm — COMMENTED** [review]
:shipit:

File: aspnetcore/migration/31-to-50.md [hunk 2]
```diff
@@ -120,3 +120,24 @@ As part of the move to ".NET" as the product name, the Docker images moved from
...
+
+### DateTime values are model bound as UTC times
+
+In previous versions, `DateTime` values were model bound as local time where the timezone was determined by the server. DateTime values bound from input formatting (JSON), and `DateTimeOffset` values were bound as UTC timezones. In ASP.NET Core 5.0 and later, model binding consistently binds `DateTime` values with UTC timezone.
```
Comment (scottaddie) [inline aspnetcore/migration/31-to-50.md:128]:
```suggestion
In ASP.NET Core 3.1 and earlier, `DateTime` values were model bound as local time where the timezone was determined by the server. `DateTime` values bound from input formatting (JSON) and `DateTimeOffset` values were bound as UTC timezones. In ASP.NET Core 5.0 and later, model binding consistently binds `DateTime` values with the UTC timezone.
```

File: aspnetcore/migration/31-to-50.md [hunk 3]
```diff
@@ -120,3 +120,24 @@ As part of the move to ".NET" as the product name, the Docker images moved from
...
+
+To retain the previous behavior, remove the `DateTimeModelBinderProvider` as part of the application startup:
+
+```C#
```
Comment (scottaddie) [inline aspnetcore/migration/31-to-50.md:132]:
```suggestion
```csharp
```

**Review by scottaddie — CHANGES_REQUESTED** [review]

**Pull request merged**